### DISCUSSION TITLE
Reference Project

### DISCUSSION THREAD

**Discussion by Tarek-Hasan** [ivangabriele/clamav-desktop#64] [Feature Request Proposals]
First, thanks for your effort to make a ClamAV GUI for windows. After [ClamWin](https://github.com/clamwin/clamwin) project was abandoned, I started using Microsoft Defender, which is buggy and consumes too much ram. Then I moved to using free antivirus software which are better than Defender but installs many Bloatware and takes up large disk space. Also have many process running in the background from those unwanted apps.

Understandably, you can imagine my delight when I found your project. But that also quickly died down as I learnt its in very early stage. So I thought if I could help you by giving some reference to work with.
- [ClamTk](https://github.com/dave-theunsub/clamtk) is a ClamAV graphical front-end for Linux systems. It has an easy-to-use interface and provides context menu functionality for file managers. The project was started in 2004, and it still well maintained.

Hope this help you in anyway in the project's development.

**Comment by ivangabriele** [comment]
Hi @Tarek-Hasan,

The first reason that made me start ClamAV Desktop project is not because of the lack of Windows support but the fact that I tried clamtk and was quite disappointed. And it was under Ubuntu, which is my main and everyday OS. It didn't work (at least not out of the box) and lacked many useful options despite being around for so much time. That being said, I do have a lot of respect for their work. They are already referenced on ClamAV documentation. I don't think there is a single Linux users knowing ClamAV who is unaware of clamtk existence ;).

The goal of ClamAV Desktop is a lot more than just being a GUI. The GUI is just an intermediate step and the next step will be to compile ClamAV binaries internally so that it can become an all-in-one cross-OS solution as easy to install and use than Avast or BitDefender. ClamAV Desktop is for everybody, especially for people who are not techies and power users.

Now, indeed this project is taking time to progress since I'm alone, have quite a lot on my plate and there is a lot more code to write than clamtk has. But that's the "normal life" of pretty much any early-stage open source project. I have no doubt that once the first working versions are released, a few others will join :).
